docs/modules/ROOT/pages/tutorial-onpremise.adoc

```diff
@@ -1,32 +1,39 @@
-= On-premise example
-
-[abstract]
-CMOS is deployed to monitor an on-premise cluster with a simple Nginx web server exposed on port 8080 to provide a landing page.
-
-include::partial$tutorial.adoc[]
+= CMOS On Premise Deployment
 
 .Microlith overview
 image::microlith-runtime.png[]
 
-== Usage
+CMOS is deployed to monitor an on-premise cluster with a simple Nginx web server exposed on port 8080 to provide a landing page.
 
-To run refer to the xref:quickstart.adoc[quick start guide] for full details:
+== Deployment
 
+At this moment we only support deployment using docker. You can get started by deploying the docker container:
 [source, console]
 ----
-docker run --name=cmos --rm -d -P couchbase-observability-stack
+$ docker run --name=cmos --rm -d -P couchbase/observability-stack
 ----
+There are 2 ways we can access different services.
 
-Use `docker ps` or `docker inspect X` to see the local ports exposed, the mapping to `3000` is the Grafana one so to get this:
+Use `docker ps` or `docker inspect X` to see the local ports exposed, there are multiple services running inside the container and each service has it's own port. For example, the mapping to `3000` is the Grafana, to get the port of the host machine use this:
 
 [source, console]
 ----
-docker container port couchbase-grafana 3000
-0.0.0.0:55124
-:::55124
+$ docker container port cmos 3000
+----
+----
+0.0.0.0:49283
+:::49283
 ----
 
-Browse to `localhost:55124` and log in with the default credentials of `admin:password` for Grafana.
+A simple Nginx web server is exposed on port 8080, it exposes  different services in different paths. You can navigate to any service from here.
+[source, console]
+----
+$ docker container port cmos 8080
+----
+----
+0.0.0.0:49278
+:::49278
+----
 
 == Configuration options
 
```

Comment:
CMOS gives you fine grained control to the various services included. <Diagram reference> .. As shown in the diagram there are some default values ... We provide ways to override these values using environment variables and native configuration files of each component.

Continue with what is already there.

Reply:
Also, Get rid of single big snippet code about environment variable configuration and put it into each component section.